pkgs/by-name/ma/mangal/package.nix

```
@@ -30,8 +38,7 @@ buildGoModule rec {
   doCheck = false; # test fail because of sandbox
 
   meta = with lib; {
-    description =
-      "A fancy CLI app written in Go which scrapes, downloads and packs manga into different formats";
+    description = "A fancy CLI app written in Go which scrapes, downloads and packs manga into different formats";
```

```suggestion
    description = "CLI app written in Go which scrapes, downloads and packs manga into different formats";
```
removes article and subjective statement as per https://nixos.org/manual/nixpkgs/unstable/#var-meta-description